src/App.vue

```diff
@@ -2,6 +2,7 @@
   <v-app
     id="app-wrapper"
   >
+    <link href="https://cdn.jsdelivr.net/npm/@mdi/font@4.x/css/materialdesignicons.min.css" rel="stylesheet">
```

Comment:
I think these should already be bundled with the app? Definitely check out how other icons are done. We shouldn't need to import more